src/Microsoft.AspNet.JsonPatch/Helpers/JsonPropertyMetadata.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNet.JsonPatch.Helpers
```

Comment:
`.Helpers` is always wrong, we're the ones writing the code so let's factor it how we want.